### ISSUE TITLE
Requests made to the API before geostore registration is complete.

### ISSUE THREAD

**Issue by richardbarad** [wri/gfw-mapbuilder#657]
In the latest version (v1.4.2): http://blueraster.teaches-yoga.com/CMR?l=en

If I click on a shape in the map, I am immediately able to click on the analysis tab and run an analysis. This sometimes results in the user running an analysis prior to the completion of the geostore registration. In previous versions, the app included a message indicating that the shape was being registed with the API. Please make sure this is still included in this version.

**Comment by KaylaKremer** [comment]
@richardbarad ,

Here's a build to test: http://alpha.blueraster.io/gfw-mapbuilder/657-geostore-bug/

I set it up so that the message for the geometry still registering will appear in the footer of the analysis panel. And when it finishes, then the run analysis button & print report button appears. I figured this was best so that it completely prevents the user from trying to run an analysis before the geometry finishes registering. Let me know what you think!
